eng/targets/Helix.Common.props

```diff
@@ -47,7 +47,7 @@
         <HelixAvailableTargetQueue Include="$(HelixQueueArmDebian11)" Platform="Linux" />
 
         <!-- Mac -->
-        <HelixAvailableTargetQueue Include="OSX.1015.Amd64.Open" Platform="OSX" />
+        <HelixAvailableTargetQueue Include="OSX.1100.Amd64.Open" Platform="OSX" />
```

Comment:
We already test on OSX 11.0 in the regular CI. Suggest trying on 12.0 or 13.0

Reply:
Done